Mimir/GraphQL/Types/AgentType.cs

```diff
@@ -47,4 +63,54 @@ protected override void Configure(IObjectTypeDescriptor<AgentObject> descriptor)
                     .ToList();
             });
     }
+
+    private static (
+        AgentRepository agentRepo,
+        PlanetName planetName,
+        Address agentAddress)? GetSource(IResolverContext context)
+    {
+        var agentRepo = context.Services.GetService<AgentRepository>();
+        if (agentRepo is null)
+        {
+            return null;
+        }
+
+        var planetName = context.ScopedContextData.TryGetValue("planetName", out var pn)
+            ? (PlanetName?)pn
+            : null;
+        if (planetName is null)
+        {
+            return null;
+        }
```

Comment:
(I didn't test this suggestion code is compilable)

```suggestion
        if (context.ScopedContextData.TryGetValue("planetName", out var pn) && pn is not { } planetName)
        {
            return null;
        }
```

Reply:
`context.ScopedContextData.TryGetValue(key, out value)` 메서드로 얻을 수 있는 `value`의 타입은 `object?`입니다. 제안해주신 문법은 그 범위에서는 좀 더 그럴싸해 보이지만, 메서드 끝까지 함께 보면 사용하기가 어렵습니다.
대신 이렇게 수정해봤고, 잘 동작하네요.
The type of `value` we can get with the method `context.ScopedContextData.TryGetValue(key, out value)` is `object?`. The syntax you suggested makes more sense in that scope, but it's hard to use when you see it all the way through the method.
I modified it like this instead, and it works well.

```suggestion
        if (!context.ScopedContextData.TryGetValue("planetName", out var pn) ||
            pn is not PlanetName planetName)
        {
            return null;
        }
```